### PR TITLE
docs(core): adding accessability label and  removing the hardcoded background 

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/6208

#### Please provide a brief summary of this pull request.

adding accessability label and  removing the hardcoded background

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/link/examples/link-example.component.html [hunk 1]
```diff
@@ -1,18 +1,21 @@
-<a [routerLink]="['./']" fd-link>Standard Link</a> <br /><br />
-<a [routerLink]="['./']" fd-link [emphasized]="true">Emphasized Link</a>
+<a [routerLink]="['./']" fd-link aria-label="Standard link">Standard Link</a> <br /><br />
+<a [routerLink]="['./']" fd-link [emphasized]="true" aria-label="my Link">Emphasized Link</a>
```
Comment (KevinOkamoto) [inline apps/docs/src/app/core/component-docs/link/examples/link-example.component.html:2]: Shouldn't the `aria-label` be "Emphasized Link"?

Comment (Lokanathan-k) [reply apps/docs/src/app/core/component-docs/link/examples/link-example.component.html]: fixed

**Review by manu-kr — APPROVED** [review]
LGTM

**Review by N1XUS — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/link/examples/link-example.component.html [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
     <fd-icon [glyph]="arrowLeft$ | async"></fd-icon>
     Icon Left Link
 </a>
 <br /><br />
-<div style="background-color:#314a5e;padding:10px">
-    <a fd-link [routerLink]="['./']" fragment="inverted" [inverted]="true">Inverted Link</a>
-</div>
\ No newline at end of file
+
+<fd-shellbar>
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/link/examples/link-example.component.html:17]: In my opinion it's an overkill to bring the Shellbar component in the documentation. I'd suggest using exactly the same approach as in fund-styles. Also, you should not be using the inverted link inside the Shellbar logo component.

**Comment by puru-hk** [comment]
Tested Issue 1 and 2 are fixed

**Review by puru-hk — APPROVED** [review]

**Pull request merged**